app/src/main/java/org/simple/clinic/scheduleappointment/facilityselection/FacilitySelectionActivityController.kt

```diff
@@ -184,17 +180,7 @@ class FacilitySelectionActivityController @Inject constructor(
                     .associateUserWithFacility(it, facility)
                     .andThen(facilityRepository.setCurrentFacility(it, facility))
               }
-              .doOnComplete { clearAndSyncReports() }
```

Comment:
I don't really understand why the reports clearing is deleted here. Let's discuss in person?

Reply:
In case of changing facility of a user, we need to delete existing reports since the reports are generated based on a particular facility.
Whereas in this case, facility change is for a patient and not a user. Hence we do not need to delete reports.